exec/cnex/support.h

```diff
@@ -0,0 +1,17 @@
+#ifndef SUPPORT_H
+#define SUPPORT_H
+
+#include "util.h"
+
+typedef struct tagTPath {
```

Comment:
Why is this type declared in the header file? It's not used in the header file anywhere.

Reply:
Initially, the list was stored in cnex.c, but I quickly realized that I didn't need to store it there;  only in support.c.  So it got moved, this got left behind.
Fixed, now.